_posts/2021-09-07-design-of-the-koji-room-koji-muro.md

```diff
@@ -0,0 +1,461 @@
+
+Here's an example of the tent-type koji-making machine:
+
+
+![](/assets/images/koji_room/Tentschematic.png){:height="550" .center}
+
+
+![](/assets/images/koji_room/Tentopen.png){:height="550" .center}
+
+
+![](/assets/images/koji_room/Tentclosed.png){:height="550" .center}
+*Top to bottom: tent-type koji machine schematic, with the cloth tent removed, and with the cloth tent attached - from Hakuyo Co., Ltd.*
+
+Here's a simpler box-type koji-making machine, which does not recirculate any air:
+
+![](/assets/images/koji_room/Hakkomachine.png){:height="550" .center}
+*A box-type koji-making machine situated in a koji room - from Hakuyo Co., Ltd.*
+
+There are also unventilated type machines, such as [Fujiwara's VEX-type machine](https://www.fujiwara-jp.com/product/detail.php?id=32&l=en) which simply uses a thin layer of koji and a semi-permeable ePTFE membrane to control moisture removal.
+
+While it may be difficult procure Japanese koji-making equipment overseas, you can look at their product catalogue sheets to get a feel of how they're designed, their capacities, and the size of the blower. 
+
+Some other equipment worth mentioning, primarily used to reduce labour costs:
+
+#### Air shooter
+
+A common way to transfer steamed substrate into the koji room, and finished koji out of the room, is to use a device called an *air shooter*. This is a a Japanese *wasei-eigo* term for what would commonly be called a pneumatic conveying system consisting of: 1) a blower 2) a feed assembly consisting of a hopper, rotary valve, and solids-conveying educator, 3) a bunch of hose, and 4) an outlet cyclone (optional). These are much faster than transferring substrate by hand. 
+
+![](/assets/images/koji_room/airshooter.png){:height="550" .center}
+*Top: a schematic of an air-shooter (pneumatic conveying system), bottom-left: feed assembly, bottom-right: blower. From [1] and Toyo Shokai Co., Ltd.*
+
+#### Mixing (kiri-kaeshi) machine
+
+A quicker way of performing koji mixing is with a mixing (*kiri-kaeshi*) machine. This is nothing then a solids mixer with a mesh screen for breaking up clumps. 
+
+![](/assets/images/koji_room/kirikaeshimachine.png){:height="550" .center}
+*A mixing (kiri-kaeshi) machine, from Hakuyo Co., Ltd.*
+
+#### Load cells
+
+Some breweries install load cells at the feet of their koji-making table to track substrate weight throughout the process. 
+
+![](/assets/images/koji_room/loadcell.png){:height="550" .center}
+*A load cell installed underneath a table (toko) in a traditional style room, from Hakuyo Co., Ltd.*
+
+#### The room
+
+And finally here's the best picture I could find of a koji room online, which is cedar-lined, uses a modern koji-making machine, and includes wall-mounted panel heaters, a mixing machine, and some other features discussed in this post. 
+
+![](/assets/images/koji_room/Bestroom.jpg){:height="550" .center}
+*A photo of koji room from www.sake-hourai.co.jp*
```

Comment:
In the description, can you point out the different equipment?

Reply:
Added small description